backend/.pre-commit-config.yaml

```diff
@@ -18,4 +12,18 @@ repos:
       - id: check-json
       - id: check-toml
       - id: check-yaml
-      - id: requirements-txt-fixer
\ No newline at end of file
+      - id: requirements-txt-fixer
+
+  - repo: https://github.com/psf/black
+    rev: 21.12b0
+    hooks:
+      - id: black
+        language_version: python3.12
+
+  - repo: https://github.com/PyCQA/flake8
+    rev: 6.0.0
+    hooks:
+      - id: flake8
+        args:
+          - --ignore=E203, E266, E501, W503
+          - --line-length=88
```

Comment:
It's kind of a carry over from C-styling, but it's good idea to put an extra empty line at the end of a file and get rid of this red circle bb.